crates/ursa-gateway/src/cli.rs

```diff
@@ -35,4 +40,12 @@ pub struct DaemonCmdOpts {
     /// tls key path
     #[arg(long)]
     pub tls_key_path: Option<PathBuf>,
+    /// indexer cid url
+    #[arg(long)]
+    pub indexer_cid_url: Option<String>,
+    /*
+     * /// indexer mh url
+     * #[arg(long)]
+     * pub indexer_mh_url: Option<String>,
```

Comment:
I think it will be useful somehow and keep it there just to remind maybe we could leverage that someday.
> https://docs.cid.contact/query-and-retrieve/querying-indexer-provider#get-request-1